cmd/version/version.go

```diff
@@ -66,6 +64,6 @@ func NewVersionCmd() *cobra.Command {
 	}
 }
 
-func runVersion(cmd *cobra.Command, args []string) {
-	GetVersion().Print()
+func runVersion(_ *cobra.Command, _ []string) {
```

Comment:
What is the motivation of this change?

Reply:
`cmd` and `args` are not used inside the function, so no need to refer them.
https://golang.org/ref/spec#Blank_identifier